repository/Seaside-Examples.package/WASessionProtectedCounter.class/properties.json

```diff
@@ -0,0 +1,11 @@
+{
+	"commentStamp" : "",
+	"super" : "WACounter",
```

Comment:
I don't think we should encourage subclassing for this. I would rather wrap a `WACounter` or add a decoration.

Reply:
Ok. Will change